public/editor/scripts/editor/js/fc/bramble-ui-bridge.js

```diff
@@ -108,7 +108,7 @@ define(function(require) {
     // *******EVENTS
     // User bar menu help
     $("#navbar-help").click(function() {
-      window.open("https://support.mozilla.org/" + locale + "/products/webmaker/thimble");
+      window.open("https://github.com/mozilla/thimble.mozilla.org/wiki/Using-Thimble-FAQ");
```

Comment:
What if we open this in a new tab/window vs. the current window?  Add `, "_blank"` after your closing quote.